### PR TITLE
Update templates to remove classes with underscores

### PR DESCRIPTION
**Pull request by andrew-147**

### CODE REVIEW

File: templates/content/comment.html.twig [hunk 1]
```diff
@@ -0,0 +1,111 @@
+{#
+/**
+ * @file
+ * Theme override for comments.
```
Comment (T2L) [inline templates/content/comment.html.twig]: Draft's theme implementation to display comments.

File: templates/content/node.html.twig [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{#
+/**
+ * @file
+ * Theme override to display a node.
```
Comment (T2L) [inline templates/content/node.html.twig]: The same as above

**Review by T2L — CHANGES_REQUESTED** [review]

**Review by T2L — APPROVED** [review]

**Review by T2L — CHANGES_REQUESTED** [review]
Please fix accidentally approved changes

File: templates/content/comment.html.twig (hunk 1 above)
Comment (andrew-147) [reply templates/content/comment.html.twig]: Amended docblock descriptions.

File: templates/content/node.html.twig (hunk 2 above)
Comment (andrew-147) [reply templates/content/node.html.twig]: Amended docblock descriptions.

**Review by T2L — APPROVED** [review]

**Pull request merged**